packages/mobile/src/invite/saga.ts

```diff
@@ -174,6 +176,7 @@ export function* sendInvite(
     const inviteCode = createInviteCode(temporaryWalletAccount.privateKey)
```

Comment:
If we are inviting without a code, we'd like to avoid generating the temporary account.